benchmark/metrics_parser.py

```diff
@@ -0,0 +1,276 @@
+# Copyright ClusterHQ Inc.  See LICENSE file for details.
+
+import argparse
+from collections import defaultdict
+import csv
+import itertools
+import json
+
+
+WALL_CLOCK_KEY = u'-- WALL --'
+
+
+HEADERS = [
+    u'Flocker Version',
+    u'Nodes',
+    u'Containers',
+    u'Scenario',
+    u'Control Service CPU',
+    u'Dataset Agent CPU',
+    u'Container Agent CPU',
+    u'Containers Converged Within Limit',
+    u'Scenario Requests Within Limit',
```

Comment:
Can we set these titles to name the limit (30 or 60 seconds), so it is clear in the output when if we change them?